server/viame_server/viame_detection.py

```diff
@@ -128,7 +127,9 @@ def export_detections(self, folder):
 
         filename = ".".join([file["name"].split(".")[:-1][0], "csv"])
 
-        csv_string = viame.export_tracks_as_csv(file)
+        imageFiles = sorted([f['name'] for f in Folder().childItems(folder) if (f['name'].endswith('png') or f['name'].endswith('jpg') or f['name'].endswith('jpeg'))])
```

Comment:
Use types from viame_server/utils.py

Do the filtering within the database rather than in python:  see `filters` kwarg and pymongo docs.  I'm not sure what the exact syntax for that filter kwarg dict is, you might have to ask in #girder

Reply:
`filters={"lowerName": {"$regex": r"\.jpg$"}}`

Courtesy of Zach.  You'll have to modify the actual regex to match all of them